### PR TITLE
xrRender_R4: Possible fix to DX11 crash on game startup

### PR DESCRIPTION
**Pull request by ShokerStlk**
People still whine that DX11 does not work. It should fix it.
Revert [4c4fad6] - added support for creation with feature level from 9.1 to 11.1

### CODE REVIEW

File: src/Layers/xrRenderDX10/dx10HW.cpp [hunk 1]
```diff
@@ -86,29 +86,14 @@ void CHW::CreateDevice(SDL_Window* m_sdlWnd)
     HRESULT R;
 
 #ifdef USE_DX11
-    D3D_FEATURE_LEVEL featureLevels[] =
-    {
-        D3D_FEATURE_LEVEL_11_1,
+    D3D_FEATURE_LEVEL pFeatureLevels[] = {
```
Comment (Xottab-DUTY) [inline src/Layers/xrRenderDX10/dx10HW.cpp]:
```suggestion
    D3D_FEATURE_LEVEL pFeatureLevels[] =
    {
```

**Comment by Seakad** [comment]
Работает

File: src/Layers/xrRenderDX10/dx10HW.cpp (hunk 1 above)
Comment (ShokerStlk) [reply src/Layers/xrRenderDX10/dx10HW.cpp]: Done.

File: src/Layers/xrRenderDX10/dx10HW.cpp [hunk 2]
```diff
@@ -88,27 +88,13 @@ void CHW::CreateDevice(SDL_Window* m_sdlWnd)
...
-        return D3D11CreateDevice(m_pAdapter, D3D_DRIVER_TYPE_UNKNOWN,
-            nullptr, createDeviceFlags, level, levels,
-            D3D11_SDK_VERSION, &pDevice, &FeatureLevel, &pContext);
-    };
-
-    R = createDevice(featureLevels, count);
-    if (FAILED(R))
-        R = createDevice(&featureLevels[1], count - 1);
+    R = D3D11CreateDevice(m_pAdapter,
+        D3D_DRIVER_TYPE_UNKNOWN, // Если мы выбираем конкретный адаптер, то мы обязаны использовать D3D_DRIVER_TYPE_UNKNOWN.
```
Comment (FreeZoneMods) [inline src/Layers/xrRenderDX10/dx10HW.cpp:95]: Write comments in English, please

Comment (ShokerStlk) [reply src/Layers/xrRenderDX10/dx10HW.cpp:95]: Ok. It's vanila code

**Pull request merged**